### ISSUE TITLE
undefined reference to `xdg_popup_interface'

### ISSUE THREAD

**Issue by andreldm** [adlocode/xfway#2]
On Arch Linux I get this error. I have weston 6.0.1 installed and used `./autogen.sh --prefix=/usr`. When meson is used instead it complains about "libweston-4" not found.

**Comment by chkboom** [comment]
Same here with Debian 10. I've got libweston-5 installed.

**Comment by brihadeesh** [comment]
Same on Void Linux with libweston-9 installed.

**Comment by adlocode** [comment]
Yeah this is happening to me, I have no idea why. I tried adding WAYLAND_PROTOCOLS_LIBS to the Makefile.am in both the src and protocols directory, it made no difference. Surely as long as I have wayland-protocols, this shouldn't be happening?

**Comment by adlocode** [comment]
The xdg_popup_interface thing, that is.

I haven't touched the meson.build in a long time, I'm using autotools.

**Comment by adlocode** [comment]
I haven't touched the meson.build in a long time, I'm using autotools.

**Comment by Batcastle** [comment]
It looks like on Ubuntu 22.04 that you need to install `libweston-9-dev`, then either modify lines 124 and 125, or lines 126 and 127 of `configure.ac.in` to point to `libweston-9` and `libweston-desktop-9` with version `9.0.0`.

Or at least, it made the error go away when I run `./autogen.sh --PREFIX="$PWD/build"`. Not sure if it actually builds yet.

**Comment by topazus** [comment]
I also got stuck in the ```undefined reference to `xdg_popup_interface'``` issue, but I do not find a way to solve this error.
